homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -78,69 +88,6 @@ async def start_feed_manager(event: Event) -> None:
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, start_feed_manager)
```

Comment:
Note, something to keep in mind for a follow-up PR: should the feed manager not be stopped somehow on `EVENT_HOMEASSISTANT_STOP`?
From what I can see async_track_time_interval initialised on `manager.async_init` but never stopped...